js/gallery.js

```diff
@@ -0,0 +1,184 @@
+			}
+
+			else{
+				var nextImage = $(nextImageElement).attr('src');
+
+				$('.center-box > img').attr('src',nextImage);
+				updateNextImage();
+			}
+		});
+
+		// Click event for prev button
+		$(document).on('click', '.btn-prev',function(e){
+			// debugger;
+			if( $('.center-box > img').attr('src') ===  $(firstImageElement).find('img').attr('src') ){
+				e.preventDefault();
+			}
+
+			else{
+				var prevImage = $(prevImageElement).attr('src');
+
+				$('.center-box > img').attr('src',prevImage);
+				updatePrevImage();
+			}
+		});
+
+		$(document).on('mousedown', '.btn-prev',function(){
+			$(this).css({
+				'background' : 'url(images/btn-prev-active.png) no-repeat',
+				'background-size': 'cover'
+			});
+		});
+
+		$(document).on('mouseup', '.btn-prev',function(){
+			$(this).css({
+				'background' : 'url(images/btn-prev.png) no-repeat',
+				'background-size': 'cover'
+			});
+		});
+
+		$(document).on('mousedown', '.btn-next',function(){
+			$(this).css({
+				'background' : 'url(images/btn-next-active.png) no-repeat',
+				'background-size': 'cover'
+			});
+		});
+
+		$(document).on('mouseup', '.btn-next',function(){
+			$(this).css({
+				'background' : 'url(images/btn-next.png) no-repeat',
+				'background-size': 'cover'
+			});
+		});
+
+		var updateNextImage = function(){
+			prevImageElement = currentImageElement; // Copies current Image to previous image
+			currentImageElement = nextImageElement; // Copies next image to current image
+
+			nextImageElement = $(nextImageElement).parent().next().find('img');//update next image
+		}
+
+		var updatePrevImage = function(){
+			nextImageElement = currentImageElement; // Copies current Image to next image
+			currentImageElement = prevImageElement; // Copies previous Image to current image
+
+			prevImageElement = $(prevImageElement).parent().prev().find('img');//update previous image
+		}
+		
+	}
+
+// Main function
+	$.fn.imageGallery = function(){
+		var _this = this,
+		checkElement = function(evt){
+			if( !$('.center-box > img').is(evt.target) && !$('.center-box .nav-wrapper').is(evt.target) && !$('.center-box .btn-next').is(evt.target) && !$('.center-box .btn-prev').is(evt.target) )
+				return true;
+			else
+				return false;
+		};
+
+		$(_this).on('click', 'img', function(e){
+			// debugger;
+			var imgOverlay = new ImageOverlay(),	
+			imgContainer = new ImageDisplay(),
+			curImage = new CurrentImage( $(this).attr('src') ),
+			navigaitonButtons = new setNavigationButtons( e.target );
+
+			
+			function appendToMainWrapper(){
+				$(imgContainer.centerDiv).append(navigaitonButtons.prevButton);
+				$(imgContainer.centerDiv).append(curImage.currentImage);
+				$(imgContainer.centerDiv).append(navigaitonButtons.nextButton);
+
+				// console.log(e.target);
+				// debugger;
+
+				$(_this).parent().append(imgOverlay.overlay);
+				$(_this).parent().append(imgContainer.imageContainer);
+			}
+
+			function init(){
+				appendToMainWrapper();
```

Comment:
function are written to perform certain task, here you have just call the function. Instead you can directly call the function you want and skip unnecessary code like this one.